package.json

```diff
@@ -12,38 +12,44 @@
   "author": "sebworks",
   "private": true,
   "scripts": {
-    "dev": "pnpm install && pnpm i18n:create-locales-list && pnpm i18n:update-locales && nuxt --hostname 0.0.0.0",
+    "prepare": "husky install",
+    "ver": "func() { pnpm ls --depth=0 | grep -e \"^${1}\\s\" | awk '{print $2}'; }; func",
```

Comment:
This pains me _deeply_ to read. Does this need to be generic, is it used elsewhere, could it be a YAGNI to complicate things in this way? If so, could we move it to a proper shell script in `bin/get-package-version.sh` or something?

Reply:
It is a complicated expression, yes, but it's just a simple generalisation of the `playwright-version` script that existed before it:
```bash
pnpm ls --dev --depth=0 | grep playwright | awk '{print $2}'
```
It's only used in one place (https://github.com/WordPress/openverse-frontend/pull/1207/files#diff-ad76caf254756b7bb67a44577cdd651afc57e3b77289360b59868361b3a62a12R7) and that too seems more informational that functional. We could get rid of it altogether. Having a separate shell script for a one-liner would be overkill.

Reply:
It's not just informational, the environment variable is used in the docker-compose.yml to determine what version of the playwright image to pull.

We could just add the version getting function into the `playwright.sh` script instead of leaving it to pollute the package.json scripts.

I would prefer that a lot actually, it doesn't need to be in package.json at all (I think it was only in there originally because it was being used directly by another package.json script but that's no longer the case).

Reply:
Seems that we agree, moved in 8b60e2b.